doc/administrator/administrate.rst

```diff
@@ -38,67 +38,67 @@ You can associate metadata to all theme elements (tree items).
 Each metadata entry has the following attributes:
 
-* ``Item``: the tree item the metadata is associated to.
 * ``Name``: the type of ``Metadata`` we want to set (the available names are configured in the ``vars``
   files in ``admin_interface/available_metadata``).
 * ``Value``: the value of the metadata entry.
 * ``Description``: a description.
 
 To set a metadata entry, create or edit an entry in the Metadata view of the administration UI.
-Regarding effect on the referenced tree item on the client side, you will find a description for each sort of metadata in the `NGEO documentation <https://camptocamp.github.io/ngeo/master/apidoc/gmfThemes.GmfMetaData.html>`_.
+Regarding effect on the referenced tree item on the client side, you will find a description for each sort
+of metadata in the `NGEO documentation
+<https://camptocamp.github.io/ngeo/master/apidoc/gmfThemes.GmfMetaData.html>`_.
+
+.. TODO: the URL should be fixed when the ngeo documentation wil be generated.
 
 Functionalities
 ---------------
 
 ``Roles`` and ``Themes`` can have some functionalities. Attributes:
 
-* ``Name``: the type of Metadata we want to set (configured in the ``vars``
+* ``Name``: the type of functionality we want to set (configured in the ``vars``
   files in ``admin_interface/available_functionalities``).
-* ``Value``: the value of the metadata.
 * ``Description``: a description.
+* ``Value``: the value of the functionality.
 
 .. _administrator_administrate_layers:
 
 Layers
 ------
 
-Layer definition for the ngeo client is considered "version 2",
-whereas layer definition for the CGXP client is considered "version 1".
-In version 2, we split the layer table into 2 tables: ``layer_wms``,
-``layer_wmts``, and we copy the previous layer table in ``layerv1``.
-For information on migrating layers from CGXP to ngeo, see
+Layer definition will be splited into tow tables: ``layer_wms``, ``layer_wmts``.
+The table ``layerv1`` steel be here only to be able to migrate the application,
+for information on migrating layers from CGXP to ngeo, see
 :ref:`integrator_upgrade_application_cgxp_to_ngeo`.
 
-Still using the CGXP client? Then you should maintain both version
-of the layers, respectively ``layerv1`` and ``layer_wms``-``layer_wmts``.
-
-And the ``order`` will be moved in the relation of the tree.
-
 All layer type
 ~~~~~~~~~~~~~~
 
 All the layers in the admin interface have the following attributes:
 
 * ``Name``: the name of the WMS layer/group, or the WMTS layer.
-  It also used throw OpenLayers.i18n to display the name on the layers tree.
-* ``Public``: make the layer public, also it is accessible
-  throw the ``Restriction areas``.
-* ``Restrictions area``: the areas through which the user can see the layer.
-* ``Related Postgres table``: the related postgres table,
+  It is used throw the i18n tools to display the name on the layers tree.
+* ``Metadata URL``: deprecated
+
+.. TODO: remove deprected field
```

Comment:
This should not be a TODO in my opinion, because it is still there, therefore documented (as deprecated)

Reply:
I will fix that in another pull request :-)